metrics/examples/prometheus/alerting-rules.yaml

```diff
@@ -7,6 +7,14 @@ data:
     groups:
     - name: kafka
       rules:
+      - alert: RunningOutOfSpace
+        expr: kubelet_volume_stats_available_bytes < 5368709120
```

Comment:
Maybe a stupid question ... but will this alert on all disks? Or just on Kafka / Zookeeper disks? Can we make that so that this rule only applies for our disks?

Reply:
From group names I understand it does alert only at Kafka/Zk resources but I will sync up with @ppatierno

Reply:
no the group name is just artificial, it's a way to group alerts of same kind. For filtering on kafka/zookeeper we should check what labels we have for the `kubelet_volume_stats_available_bytes` metric.